pkg/clusteragent/autoscaling/workload/model/pod_autoscaler.go

```diff
@@ -30,6 +31,14 @@ const (
 	statusRetainedActions = 5
 )
 
+var autoscalingStatusConditions = telemetry.NewCounterWithOpts(
+	"autoscaling",
+	"status_condition",
```

Comment:
The generated metric name will not be very clear. We would expect something like `datadog.cluster_agent.autoscaling.autoscaler_conditions`